manifests/prometheus.yml

```diff
@@ -0,0 +1,547 @@
+          username: (( vault "secret/exodus/" meta.passthru "/bosh:admin_username"))
+        metrics:
+          environment: (( grab params.env ))
+    release: prometheus
+  - name: bosh_alerts
+    release: prometheus
+  - name: grafana
+    properties:
+      grafana:
+        dashboards: null
+        database:
+          name: grafana
+          password: (( vault meta.vault "/grafanadb:password" ))
+          port: 5432
+          type: postgres
+          user: grafana
+        prometheus:
+          dashboard_folders:
+          - files:
+            - /var/vcap/jobs/grafana_dashboards/*.json
+            name: Grafana
+          - files:
+            - /var/vcap/jobs/postgres_dashboards/*.json
+            name: PostgreSQL
+          - files:
+            - /var/vcap/jobs/prometheus_dashboards/*.json
+            - /var/vcap/jobs/bosh_dashboards/prometheus*.json
+            name: Prometheus
+          - files:
+            - /var/vcap/jobs/bosh_dashboards/bosh*.json
+            - /var/vcap/jobs/system_dashboards/bosh*.json
+            name: BOSH
+          - files:
+            - /var/vcap/jobs/system_dashboards/system*.json
+            name: System
+        security:
+          admin_password: (( vault meta.vault "/grafana:password"))
+          admin_user: admin
+          secret_key: (( vault meta.vault "/grafana:secret_key"))
+        session:
+          provider: postgres
+          provider_name: grafana
+          provider_password: (( vault meta.vault "/grafanadb:password" ))
+          provider_port: 5432
+          provider_user: grafana
+    release: prometheus
+  - name: grafana_dashboards
+    release: prometheus
+  - name: postgres_dashboards
+    release: prometheus
+  - name: prometheus_dashboards
+    release: prometheus
+  - name: bosh_dashboards
+    release: prometheus
+  - name: system_dashboards
+    release: prometheus
+  - name: postgres
+    properties:
+      databases:
+        databases:
+        - citext: true
+          name: grafana
+        port: 5432
+        roles:
+        - name: grafana
+          password: (( vault meta.vault "/grafanadb:password" ))
+    release: postgres
+  
+  - name: nginx
+    release: prometheus
+    properties:
+      nginx:
+        ssl_cert: (( vault meta.vault "/nginx/ssl_certificate:certificate" ))
+        ssl_key: (( vault meta.vault "/nginx/ssl_certificate:key" ))
+        prometheus:
+          https_port: (( grab meta.prometheus_port ))
+          http_port:  (( calc "meta.prometheus_port + 1" ))
+        grafana:
+          https_port: (( grab meta.grafana_port ))
+          http_port: 80
+        alertmanager:
+          https_port: (( grab meta.alertmanager_port ))
+          http_port:  (( calc "meta.alertmanager_port + 1" ))
+
+
+
+releases:
+- name: postgres
+  sha1: c1fcec62cb9d2e95e3b191e3c91d238e2b9d23fa
+  url: https://bosh.io/d/github.com/cloudfoundry/postgres-release?v=28
+  version: "28"
```

Comment:
Do we have to use the cloudfoundry postgres?  Or can we use our own?  Putting both on the same BOSH director will lead to some really fun shenanigans with `latest`

Reply:
I need to open an issue with the Prometheus bosh release. Although Postgres is an optional bosh link, there are some places within the templating where it's hardcoded referenced. If that can be changed, then we can use our own Postgres release. Until then, it's necessary to use the CF edition.